syntax = "proto3";

import "Common.proto";

message EnterRoomResponse
{
    int64   id = 1;
    bool    success = 2;
}

message ChatBroadcast
{
    int64   id = 1;
    string  message = 2;
}
